extensions/wikia/CuratedContent/CuratedContentHelper.class.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * CuratedContent Helper
+ */
+class CuratedContentHelper {
+	const STR_ARTICLE = 'article';
+	const STR_BLOG = 'blog';
+	const STR_FILE = 'file';
+	const STR_CATEGORY = 'category';
+	const STR_EMPTY_CATEGORY = 'emptyCategory';
+	const STR_VIDEO = 'video';
+
+	public function processLogic( $sections ) {
```

Comment:
could you rename this function to processSections or add some doc? ProcessLogic is not very descriptive.